### PR TITLE
Upgrade to Aeron 1.11

### PR DESCRIPTION
**Pull request by chbatey**
Now supports SPARC, I am sure someone asked for that :)

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -18,7 +18,7 @@ object Dependencies {
   val sslConfigVersion = "0.2.4"
   val slf4jVersion = "1.7.25"
   val scalaXmlVersion = "1.0.6"
-  val aeronVersion = "1.9.3"
+  val aeronVersion = "1.11.2"
```
Comment (patriknw) [inline project/Dependencies.scala]: is it wire compatible? (I assume so, but have to ask the mandatory question)

**Comment by chbatey** [comment]
I’ve checked before for the 1.x branch but I’ll double check

**Comment by johanandren** [comment]
Reply from Todd: 
> The protocol has always been backwards compatible. So far since 1.0

**Review by johanandren — APPROVED** [review]
LGTM, now I just need to get myself a sparc machine.

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**